pkgs/development/libraries/science/math/cudnn/extension.nix

```diff
@@ -148,22 +148,38 @@ final: prev: let
         supportedCudaVersions = [ "11.0" "11.1" "11.2" "11.3" "11.4" "11.5" "11.6" "11.7" "11.8" ];
       }
     ];
+    "8.7.0" = [
+      rec {
+        fileVersion = "10.2";
+        fullVersion = "8.7.0.84";
+        hash = "065m4bwdz3fd144gkgb08af1bpvxxsvbsbslv3av669yrylmm63d";
```

Comment:
You can only use hash with SRI hashes. Please clear the value, then try to build the package and it will give you an SRI hash with recent nix versions.

Reply:
Just pushed the fix. Is this also the case for elsewhere? For example, in my PR to bump CUDA 12.0.0 to 12.0.1: https://github.com/NixOS/nixpkgs/pull/215229.